FAIL
Kernel panic in run: PANIC at ../../threads/synch.c:198 in lock_acquire(): assertion `!lock_held_by_current_thread (lock)' failed.
Call stack: 0x800421877d 0x800420aa7c 0x800420c421 0x8004221949 0x8004221de9 0x8004221d01 0x800421d645 0x80042095b5 0x80042099d3 0x8004219038 0x800420c513 0x8004221fd4 0x800421c0a4 0x80042077bc
Translation of call stack:
0x000000800421877d: debug_panic (lib/kernel/debug.c:32)
0x000000800420aa7c: lock_acquire (threads/synch.c:201)
0x000000800420c421: malloc (threads/malloc.c:118)
0x0000008004221949: spt_find_page (vm/vm.c:90)
0x0000008004221de9: vm_claim_page (vm/vm.c:244)
0x0000008004221d01: vm_try_handle_fault (vm/vm.c:217)
0x000000800421d645: page_fault (userprog/exception.c:148)
0x00000080042095b5: intr_handler (threads/interrupt.c:352)
0x00000080042099d3: intr_entry (threads/intr-stubs.o:?)
0x0000008004219038: list_pop_front (lib/kernel/list.c:254)
0x000000800420c513: malloc (threads/malloc.c:139)
0x0000008004221fd4: supplemental_page_table_copy (vm/vm.c:296)
0x000000800421c0a4: __do_fork (userprog/process.c:198)
0x00000080042077bc: kernel_thread (threads/thread.c:485)
